### ISSUE TITLE
Getxattr should return ENOSYS instead of ENOTSUP?

### ISSUE THREAD

**Issue by HeikoSchlittermann** [bazil/fuse#254]
Today I observed that the kernel issues lots of Getxattr requests to a fuse filesystem if the Getxattr() method returns ENOTSUP (which is the default implemented in fs/fuse.go).

If this method would return ENOSYS, the kernel seems to understand this and doesn't issue further Getxattr() requests.

Kernel: 5.6.15

**Comment by tv42** [comment]
ENOSYS would mean kernel stops sending Getxattr requests. Putting that in as the default would effectively stop you from implementing xattrs on some nodes. I don't think this should be changed.

**Comment by tv42** [comment]
Yeah, I just don't see this as desirable. The typical implementation has different types for different kinds of nodes (dir/file, application-specific types). One of them not implementing Getxattr is not a reason to prevent others from implementing it, and ENOSYS means the filesystem as a whole doesn't implement an operation.
